packages/tofn/tofn.0.1/opam

```diff
@@ -0,0 +1,20 @@
+opam-version: "2.0"
+synopsis: "Typed ordered fuzzy numbers"
+maintainer: "Matthew Kukla <matt.kukla@yandex.ru>"
+authors: "Matthew Kukla <matt.kukla@yandex.ru>"
+bug-reports: "Matthew Kukla <matt.kukla@yandex.ru>"
+license: "GPL-3.0"
+homepage: "https://mkukla.net/doc/tofn/"
+dev-repo: "git+https://github.com/matt-kukla/tofn.git"
+depends: [
+  "ocaml"
+  "dune" {>= "2.9"}
+]
+build: [["dune" "build" "-p" name "-j" jobs "@install" ]]
+url {
+  src: "https://github.com/matt-kukla/tofn/archive/0.11.tar.gz"
```

Comment:
Is this expected? Should this package be version `0.11` instead?

Reply:
Also the `dune-project file and opam files got moved to the `src` directory in this tag so installing will not do anything.

Reply:
Yes, you're correct, it should be 0.11 instead, I had forgotten to update the version number.  Thanks.

Reply:
> Also the `dune-project` file and opam files got moved to the `src` directory in this tag so installing will not do anything.

Also correct, I've screwed up a number of things with this release.  Would it be easier if I opened a new pull request with the updated (and properly placed) dune/opam files, as version 0.12?

Reply:
yes, that'd be good. Could you also return https://github.com/ocaml/opam-repository/pull/19624/commits/ae7957f6915446607838cfd747c75a160722e269 upstream?